packages/frontend/src/features/videos/page/Movies.tsx

```diff
@@ -43,7 +44,7 @@ const Movies: React.SFC = () => {
     <>
       <Header/>
       <Main>
-        <VideoPlayerComponent />
+        <VideoPlayerComponent home={true}/>
```

Comment:
Co to jest to home?

Reply:
no bo ten player używany jest na różnych routach a zakładam że tylko na stronie głównej ma startować od razu